test/scripts/generate-test-app.js

```diff
@@ -76,22 +74,24 @@ yargs
       });
     },
     (argv) => {
-      const { database, run, appPath = 'test-apps/base', template } = argv;
+      const { databaseName, run, appPath = 'test-apps/base', template } = argv;
+
+      if (databaseName) {
```

Comment:
Is there a reason we're renaming `database` to `databaseName` and switching it to have priority over `dbclient`? 

No real objection, just worried it could unintentionally affect some other CI process, especially since it wasn't documented to begin with.